linter.py

```diff
@@ -25,13 +26,13 @@ class Flow(Linter):
     version_requirement = '>= 0.1.0'
     regex = r'''(?xi)
         # Warning location and optional title for the message
-        ^.+/(?P<file_name>[^/]+\.(js|html)):(?P<line>\d+):(?P<col>\d+),\d+:\s*(?P<message_title>.+)$\r?\n
+        ^.+\/(?P<file_name_1>.+):(?P<col_1>(\d+:\d+,(\d+:)?\d+)):\s(?P<message_title>.+)?\r?\n
 
-        # Main lint message
-        ^(?P<message>.+)$
+        # (Optional) main message
+        (^(?P<message>.+))?
```

Comment:
As I said, it doesn't make sense to mark a line as an error if there is no info about why flow thinks its an error. Could you please post an example of flow flagging a line with out giving a message? If that's happening an issue should probably be opened over at the flow project.